src/en/charms-storage.md

```diff
@@ -226,22 +246,27 @@ juju deploy postgresql --attach-storage pgdata/0
 !!! Note:
     The `--attach-storage` and `-n` flags cannot be used together.
 
-Add a new unit of the postgresql application, attaching existing storage
-'pgdata/0' to the new unit:
+To add a new Ceph OSD unit with (detached) existing storage 'osd-devices/2':
 
 ```bash
-juju add-unit postgresql --attach-storage pgdata/0
+juju add-unit ceph-osd --attach-storage osd-devices/2
 ```
 
-Destroy already detached storage 'pgdate/0' (remove it from the model):
+To destroy already detached storage 'osd-devices/3' (remove it from the model):
 
 ```bash
-juju remove-storage pgdata/0
+juju remove-storage osd-devices/3
 ```
 
-If an attempt is made to remove storage that is currently in use (i.e. it is
-attached) Juju will return an error.
+To upgrade the OSD journal of Ceph unit 'ceph-osd/0' from magnetic to solid
+state (SSD) and remove the uneeded original journal 'osd-journals/0':
 
+```bash
+juju add-storage ceph-osd/0 osd-journals=ebs-ssd,8G,1
```

Comment:
Does this mean that you can 'swap out' existing storage simply by adding new storage to the same slot? This would warrant further explanation I think, though I guess it depends on the charm and how it is using the storage